cli/js/web/body.ts

```diff
@@ -157,44 +150,36 @@ export class Body implements domTypes.Body {
     }
   }
 
-  public async text(): Promise<string> {
+  async text(): Promise<string> {
     if (typeof this._bodySource === "string") {
       return this._bodySource;
     }
-
-    const ab = await this.arrayBuffer();
-    const decoder = new TextDecoder("utf-8");
-    return decoder.decode(ab);
+    return new TextDecoder().decode(await this.arrayBuffer());
   }
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public async json(): Promise<any> {
-    const raw = await this.text();
-    return JSON.parse(raw);
+  async json(): Promise<any> {
+    return JSON.parse(await this.text());
   }
 
-  public arrayBuffer(): Promise<ArrayBuffer> {
+  async arrayBuffer(): Promise<ArrayBuffer> {
+    const encoder = new TextEncoder();
     if (isTypedArray(this._bodySource)) {
-      return Promise.resolve(this._bodySource.buffer as ArrayBuffer);
+      return this._bodySource.buffer;
     } else if (this._bodySource instanceof ArrayBuffer) {
-      return Promise.resolve(this._bodySource);
+      return this._bodySource;
     } else if (typeof this._bodySource === "string") {
-      const enc = new TextEncoder();
-      return Promise.resolve(
-        enc.encode(this._bodySource).buffer as ArrayBuffer
-      );
+      return encoder.encode(this._bodySource).buffer;
     } else if (this._bodySource instanceof ReadableStreamImpl) {
-      return bufferFromStream(this._bodySource.getReader());
+      const buf = await bufferFromStream(this._bodySource.getReader());
```

Comment:
This await is totally unnecessary.  Just return the value.  It causes an extra promise to be created. 😢